terraform/examples/daos_cluster/terraform.tfvars.perf.example

```diff
@@ -26,11 +26,19 @@ server_daos_scm_size       = 45
 # server_os_disk_type        = "pd-ssd"
 
 # pools = [
-#  {
-#    pool_name = "test_pool"
-#    pool_size = "1TB"
-#    containers = [ "test_container1", "test_container2" ]
-#  }
+#   {
+#     label = "test_pool"
+#     size = "1TB"
+#     tier_ratio = 3
+#     properties = "reclaim:disabled"
+#     containers = [
+#       {
+#         label = "test_cont",
+#         type = "POSIX",
+#         properties = "rf:0"
```

Comment:
Right here we need to be able to support an array of user attributes on containers.  At a minimum we need a user attribute named "automount". 

```
user_attributes = [
  automount = true,
  foo=bar
]
```